tests/basic_api_test.sh

```diff
@@ -146,3 +146,22 @@ curl -L -X POST "http://$QDRANT_HOST/collections/test_collection/points/search"
       "vector": [0.2, 0.1, 0.9, 0.7],
       "top": 3
   }' | jq
+
+# test double forward slash handling
+curl -L -X POST "http://$QDRANT_HOST//collections/test_collection/points/search" \
```

Comment:
```suggestion
curl -L -X POST "http://$QDRANT_HOST//collections/test_collection/points/search/" \
```

I believe it should work like this as well

Reply:
Sure, i've added it too